Camera2SlowMotion/app/src/main/java/com/example/android/camera2/slowmo/fragments/CameraFragment.kt

```diff
@@ -308,7 +308,8 @@ class CameraFragment : Fragment() {
                     // Prevents screen rotation during the video recording
                     requireActivity().requestedOrientation =
                             ActivityInfo.SCREEN_ORIENTATION_LOCKED
-
+                    CameraActivity.output.write("started capture")
```

Comment:
We probably want something a little more structured than raw text.  Proto is probably best, but JSON would be much easier to set up and almost as good.

Reply:
Will fix in a the following commits